src/metabase/models/recent_views.clj

```diff
@@ -298,19 +342,41 @@
                                               [:= :rc.id :rv.model_id]]]
                                  :order-by [[:rv.timestamp :desc]]}))
 
-(defn- post-process [recent-view]
+(mu/defn ^:private model->return-model [model :- :keyword]
+  (if (#{:question} model) :card model))
```

Comment:
nit, but I don't really understand why this pattern would be preferable to a straight `=` when you're only comparing against one value?

Reply:
That's fair, I didn't fix this yet, but I will now.